pkg/agent/vhd/cache/types.go

```diff
@@ -1,16 +1,14 @@
 package cache
 
-// CachedOnVHD represents the cached components on the VHD.
+// OnVHD represents the cached components on the VHD.
 type OnVHD struct {
```

Comment:
@AlisonB319 do we still need this stuff? I don't think we ever ended up using it for NIC did we?

Reply:
I think there is leftover stuff from api.GetCachedVersionsOnVHD that I will create a new PR to remove

Reply:
removed in a separate PR